clients/java/src/test/java/io/camunda/zeebe/client/impl/oauth/OAuthCredentialsCacheTest.java

```diff
@@ -36,8 +36,11 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
 import org.junit.rules.TemporaryFolder;
 
+@Disabled(
```

Comment:
:x: This is mixing junit4 and junit5 (aka "jupiter"). The test is written in junit4, the right annotation is `org.junit.Ignore`.

:x: AFAIK, the only test that consistently fails is `shouldBeThreadSafe`, so we should not disable all tests here, just this one.

Reply:
> This is mixing junit4 and junit5

💡 Today I learned.